lib/icalendar/event.ex

```diff
@@ -32,7 +32,7 @@ defimpl ICalendar.Serialize, for: ICalendar.Event do
 
   defp to_kv({key, raw_value}) do
     name  = key |> to_string |> String.upcase
-    value = Value.to_ics( raw_value )
-    KV.build( name, value )
+    value = Value.to_ics(raw_value)
+    KV.build(name, value, raw_value)
```

Comment:
I find how we're effectively passing in the value twice here a bit off. It seems redundant, and it makes the function signature harder to read.

Should the KV.build function be responsible for calling Value.to_ics?